pkg/api/controllers/projectconfig/project_config.go

```diff
@@ -75,6 +76,11 @@ func GetDefaultProjectConfig(ctx *gin.Context) {
 		statusCode := http.StatusInternalServerError
 		if config.IsProjectConfigNotFound(err) {
 			statusCode = http.StatusNotFound
+			ctx.AbortWithStatus(statusCode)
+			go func() {
```

Comment:
Why is this log inside a goroutine?

Also, please use `logrus` as we do in the rest of the codebase (e.g. https://github.com/daytonaio/daytona/blob/main/pkg/cmd/apikey/list.go#L12) and write it to the `Debug` level.

Reply:
OK. sorry for the go routine part

Reply:
Done requested changes.